mbed_host_tests/host_tests_toolbox/host_functional.py

```diff
@@ -97,13 +97,16 @@ def handle_send_break_cmd(port,
     if len(port_config) == 2:
         # -p COM4:115200
         port = port_config[0]
-        baudrate = int(port_config[1])
+        baudrate = int(port_config[1]) if not baudrate else baudrate
     elif len(port_config) == 3:
         # -p COM4:115200:0.5
         port = port_config[0]
-        baudrate = int(port_config[1])
+        baudrate = int(port_config[1]) if not baudrate else baudrate
         timeout = float(port_config[2])
 
+    if not baudrate:
+        baudrate = 115200
```

Comment:
@mazimkhan I'm not sure where I would put it in `mbed_host_tests.__init__.py`, as there are no constants defined there